storage/src/mmr/journaled.rs

```diff
@@ -85,7 +85,7 @@ impl<B: Blob, E: RStorage<B>, H: Hasher> Mmr<B, E, H> {
             let item = journal.read(i).await?;
             vec.push(item);
```

Comment:
Recommend calling `vec` something more descriptive.

Reply:
have reworked this code to bootstrap in the "forget_all" state, with more descriptive names.